# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vlock/link-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Compiler conventions... need libs at the end while linking
	-- jsaw

--- ./Makefile.orig	1999-01-14 01:19:14.000000000 +0100
+++ ./Makefile	2006-01-01 14:46:52.000000000 +0100
@@ -4,11 +4,12 @@
 # remove the -DUSE_PAM, -ldl, and -lpam if you aren't using PAM
 RPM_OPT_FLAGS=-O2
 CFLAGS = $(RPM_OPT_FLAGS) -DUSE_PAM
-LDFLAGS = -ldl -lpam -lpam_misc
+LIBS = -ldl -lpam -lpam_misc
 
 OBJS = vlock.o signals.o help.o terminal.o input.o
 
 vlock: $(OBJS)
+	$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 vlock.man: vlock.1
 	groff -man -Tascii vlock.1 > vlock.man
